app/scenes/Settings/components/UserListItem.tsx

```diff
@@ -13,34 +14,31 @@ type Props = {
   showMenu: boolean;
 };
 
-@observer
-class UserListItem extends React.Component<Props> {
-  render() {
-    const { user, showMenu } = this.props;
+const UserListItem = ({ user, showMenu }: Props) => {
+  const { t } = useTranslation();
 
-    return (
-      <ListItem
-        title={<Title>{user.name}</Title>}
-        image={<Avatar src={user.avatarUrl} size={32} />}
-        subtitle={
-          <>
-            {user.email ? `${user.email} · ` : undefined}
-            {user.lastActiveAt ? (
-              <>
-                Active <Time dateTime={user.lastActiveAt} /> ago
-              </>
-            ) : (
-              "Invited"
-            )}
-            {user.isAdmin && <Badge primary={user.isAdmin}>Admin</Badge>}
-            {user.isSuspended && <Badge>Suspended</Badge>}
-          </>
-        }
-        actions={showMenu ? <UserMenu user={user} /> : undefined}
-      />
-    );
-  }
-}
+  return (
+    <ListItem
+      title={<Title>{user.name}</Title>}
+      image={<Avatar src={user.avatarUrl} size={32} />}
+      subtitle={
+        <>
+          {user.email ? `${user.email} · ` : undefined}
+          {user.lastActiveAt ? (
+            <>
+              Active <Time dateTime={user.lastActiveAt} /> ago
```

Comment:
Needs translating, you'll need to use the `<Trans>` component with this one to insert the <Time> component in the right place… see other examples in the codebase.

Reply:
Found exactly what I was looking for https://github.com/outline/outline/blob/04f5b08ba18c50a670cdab946b49f03dfc412d71/app/scenes/Document/components/Notices.tsx#L23-L26